src/janus-controller/service/service.go

```diff
@@ -40,26 +41,24 @@ func (s *Service) Init(serverAgentIp string) {
 	schemaId := "EZpfyRHcXuohyTvbgsrg7S:2:janus-sensors:1.0"
 
 	s.ServerClient = acapy.NewClient(fmt.Sprintf("http://%s:8002", serverAgentIp))
-	_, err = helper.TryUntilNoError(s.ServerClient.Status, 20)
-	if err != nil {
-		log.Fatal("Timeout when trying to connect with issuer aca-py agent")
-	}
+	helper.TryUntilNoError(s.ServerClient.Status, 600)
 
 	// create cred definition
+	log.InfoLogger("Issuing credential definition")
 	s.CredDefinitionId, err = agents.GetCredDef(s.ServerClient, schemaId)
 
 	if err != nil {
 		if err.Error() == "empty" {
 			s.CredDefinitionId, err = agents.CreateCredDef(s.ServerClient, schemaId)
```

Comment:
i thing this would fit better

```suggestion
	log.InfoLogger("Looking for a valid credential definition")
	s.CredDefinitionId, err = agents.GetCredDef(s.ServerClient, schemaId)

	if err != nil {
		if err.Error() == "empty" {
		        log.InfoLogger("No previously created credential definition found. Issuing a new one",)
			s.CredDefinitionId, err = agents.CreateCredDef(s.ServerClient, schemaId)
```